docs/temporary_locations.md

```diff
@@ -57,12 +57,12 @@ The holding id for each item is the permanent location holding id '2293974893000
 ```
 In Orangelight in order to display the status we match the holding_id from the availability response with the holding id (hash key) from the indexed 'holdings_1display' field.  
 
-Currently if a staff member moves an item from a temporary location to a permanent location or the opposite, the new record will not get indexed immediately. See schedule for incremental indexing https://github.com/pulibrary/bibdata/blob/main/docs/alma_publishing_jobs_schedule.md. This results in a mismatch between the holding id in the JS availability response that Orangelight receives from bibdata and the indexed 'holdings_1display' holding id (hash key). Finally this results in an empty availability status in the record page. This is fixed with the next incremental job.
+Currently if a staff member moves an item from a temporary location to a permanent location or the opposite, the new record will not get indexed immediately. See [schedule](https://github.com/pulibrary/bibdata/blob/main/docs/alma_publishing_jobs_schedule.md) for incremental indexing. This results in a mismatch between the holding id in the JS availability response that Orangelight receives from bibdata and the indexed 'holdings_1display' holding id (hash key). Finally this results in an empty availability status in the record page. This is fixed with the next incremental job.
 
 ## Case 2: 
 
 Temporary location "RES_SHARE$IN_RS_REQ". This is a **unique** temporary location case.
-When an item is moved to this specific temporary location we index it as a permanent location. The JS availability for this item would be: https://bibdata.princeton.edu/bibliographic/availability.json?deep=true&bib_ids=995217553506421 
+When an item is moved to this specific temporary location we index it as a permanent location. The [JS availability](https://bibdata.princeton.edu/bibliographic/availability.json?deep=true&bib_ids=995217553506421) for this item would be:  
```

Comment:
Can we change the link text to [JS availability response] similar to the previous one you updated?